### ISSUE TITLE
[a11y] Add another should be removed for tree definition items

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1311]
![image](https://user-images.githubusercontent.com/37256050/164738648-e6f7c77a-9f05-460d-a0b2-e484537190c0.png)

`/specify/view/geographytreedefitem/new/`

**Comment by maxxxxxdlp** [comment]
Fixed in [88bdff7](https://github.com/specify/specify7/commit/88bdff752b6d69ebd4ae65e2a020080f49b1037b)
